requirements.txt

```diff
@@ -1,3 +1,3 @@
-protobuf>=4.25.1
-zeroconf>=0.25.1
-casttube>=0.2.0
+casttube==0.2.1
+protobuf==4.25.2
+zeroconf==0.131.0
```

Comment:
I'm not sure if we want to pin to latest or to minimum required version when testing?

Reply:
For development, the idea is to pin strictly to the latest working versions. For production, the idea is to require a minimum version as the oldest working version.

Depending on the situation a maximum version can also be added for production requirements (normally the next major version). Production requirements should be as forgiving as possible.